src/main/java/org/folio/circulation/resources/CirculationRulesResource.java

```diff
@@ -70,16 +70,16 @@ public CirculationRulesResource(String rootPath, HttpClient client) {
    * Register the path set in the constructor.
    * @param router  where to register
    */
+  @Override
   public void register(Router router) {
     router.put(rootPath).handler(BodyHandler.create());
-
     router.get(rootPath).handler(this::get);
     router.put(rootPath).handler(this::put);
   }
 
   private void get(RoutingContext routingContext) {
     final WebContext context = new WebContext(routingContext);
-    final Clients clients = Clients.create(context, client);
+    final Clients clients = Clients.create(routingContext, client);
```

Comment:
Could you please overload the `create` method to `create(WebContext, HttpClient)` instead?

Reply:
I think the changes I've submitted have resolved this. Please re-open this comment if not.